src/main/scala/goatrodeo/util/ArtifactWrapper.scala

```diff
@@ -73,14 +75,22 @@ object ArtifactWrapper {
     *   -- the name of the file
     */
   def mimeTypeFor(rawData: InputStream, fileName: String): String = {
-
-    val data = new BufferedInputStream(rawData)
-
-    val metadata = new Metadata()
-    metadata.set(TikaCoreProperties.RESOURCE_NAME_KEY, fileName)
-    val detected = tika.getDetector.detect(data, metadata)
-
-    detected.toString()
+    try {
+      val data = new BufferedInputStream(rawData)
+
+      val metadata = new Metadata()
+      metadata.set(TikaCoreProperties.RESOURCE_NAME_KEY, fileName)
+      val detected = tika.getDetector.detect(data, metadata)
+
+      detected.toString()
+    } catch {
+      case e: Exception =>
+        logger.error(
+          f"Tika failed, ${e.getMessage()}. Returing octet stream",
```

Comment:
Typo, and I think we can just be literal.

```suggestion
          f"Tika failed, ${e.getMessage()}. Returning application/octet-stream",
```